src/commands/misc/help.js

```diff
@@ -1,16 +1,24 @@
-const { EmbedBuilder } = require('@discordjs/builders');
+const {SlashCommandBuilder,  EmbedBuilder } = require('@discordjs/builders');
 
 module.exports = {
-    name: 'help',
-    description: 'List of commands',
+    data: new SlashCommandBuilder()
+        .setName('help')
+        .setDescription('List of commands'),
 
-    callback: (client, interaction) => {
+    options:
```

Comment:
Again doesn't have to be this PR, but dynamic commands would be helpful

Reply:
yeah, Travis and I were talking about potential future refactors, and adding more dynamic elements would be ideal. I think in another refactor.